sklearn/model_selection/_plot.py

```diff
@@ -0,0 +1,454 @@
+        else:
+            train_scores, test_scores = self.train_scores, self.test_scores
+
+        if std_display_style not in ("errorbar", "fill_between", None):
+            raise ValueError(
+                f"Unknown std_display_style: {std_display_style}. Should be one of"
+                " 'errorbar', 'fill_between', or None."
+            )
+
+        if score_type not in ("test", "train", "both"):
+            raise ValueError(
+                f"Unknown score_type: {score_type}. Should be one of 'test', "
+                "'train', or 'both'."
+            )
+
+        if score_type == "train":
+            scores = {"Training metric": train_scores}
+        elif score_type == "test":
+            scores = {"Testing metric": test_scores}
+        else:  # score_type == "both"
+            scores = {"Training metric": train_scores, "Testing metric": test_scores}
+
+        if std_display_style in ("fill_between", None):
+            # plot the mean score
+            if line_kw is None:
+                line_kw = {}
+
+            self.lines_ = []
+            for line_label, score in scores.items():
+                self.lines_.append(
+                    *ax.plot(
+                        self.train_sizes,
+                        score.mean(axis=1),
+                        label=line_label,
+                        **line_kw,
+                    )
+                )
+            self.errorbar_ = None
+            self.fill_between_ = None  # overwritten below by fill_between
+
+        if std_display_style == "errorbar":
+            if errorbar_kw is None:
+                errorbar_kw = {}
+
+            self.errorbar_ = []
+            for line_label, score in scores.items():
+                self.errorbar_.append(
+                    ax.errorbar(
+                        self.train_sizes,
+                        score.mean(axis=1),
+                        score.std(axis=1),
+                        label=line_label,
+                        **errorbar_kw,
+                    )
+                )
+            self.lines_, self.fill_between_ = None, None
+        elif std_display_style == "fill_between":
+            if fill_between_kw is None:
+                fill_between_kw = {}
+            default_fill_between_kw = {"alpha": 0.5}
+            fill_between_kw = {**default_fill_between_kw, **fill_between_kw}
+
+            self.fill_between_ = []
+            for line_label, score in scores.items():
+                self.fill_between_.append(
+                    ax.fill_between(
+                        self.train_sizes,
+                        score.mean(axis=1) - score.std(axis=1),
+                        score.mean(axis=1) + score.std(axis=1),
+                        label=line_label,
+                        **fill_between_kw,
+                    )
+                )
+
+        score_name = self.score_name if score_name is None else score_name
```

Comment:
This will output the string "None" when `score_name is None` (see the rendered document [here](https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/a2c0602d-95fa-456b-b93e-215ab855b335/1a705171-a133-4e69-ade0-209ba329c027/0/doc/_images/sklearn-model_selection-LearningCurveDisplay-1.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEONCBDMLOQ%2F20221027%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20221027T143925Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEC8aCXVzLWVhc3QtMSJHMEUCICWu2j7Wneac352TghCdn4BsT5OWrszsCF%2Fa5qbreYD6AiEAzDC3trnfVbNwC92E8lYAceph3401R9AxGuYGDx9h0g8qqwIIGBADGgwwNDU0NjY4MDY1NTYiDAWQ1Tz8SyBbPWCwKyqIAgO6hUNGAvsqmRrJgTMxkLCGL4sQgoJyeRqsB5E4U4OBgp9OUQDNxf3afBMtf3hxPGVzkyHkiT3jEnfZU4qeZXCw3SFdVccvQ3cO4u8ar1AgpUTZqJ78O4Liii3aAt5z6zmk5z80amhLgxP8nYMiFJuNgUwTYY0QNe3IWVVmSbpKcufi27lWTFdRYu0gAyFXYpYqFbhtO5%2Fz%2F%2FbBi1%2BdLApQGfluIFNY2E3RxxzaVQo8yl9Hr7GX9525U9BZ80aRMvSrvIHWCDzzOh1b3X6OGuTEVEWRelHB25ng4%2FY9SCozi5FLemRJmZo3hmBt2O61a8h2a0uqeTem5HVwpliWX2Sz7Y6x9M6ZNzDTrOqaBjqdAfQf6Mwu00y8oLnEvviUSWK5UmMbzEdhkb1QH39J9o3l6vrLAJQFGW4Xy%2FbAUZoeT3XVacpz2CyJgTIYEPwPMCbg9LTxLXNCTiaak1thPIbU%2BbCQGSNCdMrgbVYBa2yj5ZF%2FE2a8xdPKDqSNIF05StgWe3%2F7pdzE9zrwppS%2FSMsWX9YAKEqEoD%2Fm%2F7WJm%2Br%2FuFq4gOp93mbEqhFDpzU%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=9e14bd2d63898033a233c240ab35c23f6fcd6d972ab820872508171fa7443381)), not the generic text "Score" as in the `from_estimator` method.

Reply:
Uhm, something is wrong then.

Reply:
Uhm I cannot reproduce. @ArturoAmorQ could you tell me in which file did you observe this behaviour? Since it is while, the artefact is not existing anymore.

Reply:
In the first figure of [modules/generated/sklearn.model_selection.LearningCurveDisplay.html](https://output.circle-artifacts.com/output/job/c57d4315-7b0c-47c5-8525-3abfeb0c427a/artifacts/0/doc/modules/generated/sklearn.model_selection.LearningCurveDisplay.html).

Reply:
thanks